packages/@expo/cli/src/lint/index.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
```

Comment:
The shebang here isn't necessary, as this file is not the direct entry point. That's only required for **bin/cli.ts**.

```suggestion
import chalk from 'chalk';
```